src/lib/discord/commando/commands/tracked.js

```diff
@@ -119,18 +119,18 @@ exports.run = async (client, msg, command) => {
 				let genderText = ''
 				let typeText = ''
 				if (!invasion.gender || invasion.gender === '') {
-					genderText = 'Gender: any, '
+					genderText = client.translator.translate('any')
 				} else if (invasion.gender === 1) {
-					genderText = 'Gender: male, '
+					genderText = client.translator.translate('male')
 				} else if (invasion.gender === 2) {
-					genderText = 'Gender: female, '
+					genderText = client.translator.translate('female')
 				}
 				if (!invasion.grunt_type || invasion.grunt_type === '') {
-					typeText = 'Any'
+					typeText = client.translator.translate('Any')
```

Comment:
we could swap to lowercase to be consistent with other files using "any"

Reply:
Done